l10n_ch_account_statement_base_import/parsers/raiffeisen_file_parser.py

```diff
@@ -0,0 +1,285 @@
+
+    def file_is_known(self):
+        """Predicate the tells if the parser can parse the data file
+
+        :return: True if file is supported
+        :rtype: bool
+        """
+
+        return ('Booked At' in self.rows[0] and 'Text' in self.rows[0])
+
+    def _parse_currency_code(self):
+        """Parse file currency ISO code
+
+        :return: the currency ISO code of the file eg: CHF
+        :rtype: string
+        """
+
+        return 'CHF'
+
+    def _parse_statement_balance(self):
+        """Parse file start and end balance
+
+        :return: Tuple with the file start and end balance
+        :rtype: float
+        """
+
+        first_balance = float(self.rows[0].get('Balance').replace("'", ''))
+        first_amount = float(self.rows[0].get(
+            'Credit/Debit Amount').replace("'", ''))
+        balance_start = float(first_balance - first_amount)
+
+        i = 1
+        while not self.rows[-i].get('Balance'):
+            i += 1
+        balance_end = float(self.rows[-i].get('Balance'))
+
+        return balance_start, balance_end
+
+    def _parse_transactions(self):
+        """Parse bank statement lines from file
+        list of dict containing :
+            - 'name': string (e.g: 'KBC-INVESTERINGSKREDIET 787-5562831-01')
+            - 'date': date
+            - 'amount': float
+            - 'unique_import_id': string
+            -o 'account_number': string
+                Will be used to find/create the res.partner.bank in odoo
+            -o 'note': string
+            -o 'partner_name': string
+            -o 'ref': string
+
+        :return: a list of transactions
+        :rtype: list
+        """
+
+        transactions = []
+        id = 0
+        for row in self.rows:
+            transactions.append(self.get_st_line_vals(row, id))
+            id += 1
+
+        return transactions
+
+    def _get_values(self, line, currency='CHF', rate=1):
+        name = ''
+        amount = '0.0'
+        vals = line.get('Text').rsplit(' ' + currency + ' ', 1)
+        if len(vals) == 2:
+            name = vals[0]
+            amount = float(vals[1].replace("'", "")) * rate
+        else:
+            raise exceptions.Warning(
+                'ParsingError', _('Unable to parse amount for ligne %s') %
+                line.get('Text'))
+        return name, str(amount)
+
+    def get_st_line_vals(self, line, id):
+        """
+        This method must return a dict of vals that can be passed to create
+        method of statement line in order to record it.
+            :param:  line: a dict of vals that represent a line of
+                           result_row_list
+            :return: dict of values to give to the create method of
+                     statement line,
+        """
+
+        # We try to extract a BVR reference
+        result = re.match(r'.*(\d{27}).*', line.get('Text'))
+        ref = '/'
+        test = 'Cr�dit'.decode('iso-8859-15').encode('utf8')

```

Comment:
Souldn't it be Crédit instead of Crèdit?

And you might want to do `u'Crédit'.encode('utf8')` if you want a str with `utf8` output or `u'Crédit'.encode('iso-8859-15')` if you need some other iso encoding. You should better start with unicode